js/lesson3/tutorial.md

```diff
@@ -426,7 +426,7 @@ method to add something to the start of an element, and something like
 this to make the new box:
 
 ```js
-"<div class='item' style='background-color: " + color + ";'></div>"
+"<div class="item" style="background-color: " + color + ";"></div>"
```

Comment:
GJ converting HTML quotes to double quotes but I think you missed the JS quotes. It should be:
`'<div class="item" style="background-color: ' + color + ';"></div>'`